src/main/java/hudson/plugins/plot/PlotReport.java

```diff
@@ -43,16 +42,16 @@
      */
     private String group;
 
-    public PlotReport(AbstractProject<?, ?> project, String group,
+    public PlotReport(Job<?, ?> job, String group,
             List<Plot> plots) {
         Collections.sort(plots);
         this.plots = plots;
         this.group = group;
-        this.project = project;
+        this.project = job;
     }
 
     // called from PlotReport/index.jelly
-    public AbstractProject<?, ?> getProject() {
+    public Job<?, ?> getProject() {
```

Comment:
As mentioned in another case, potential problem change if any plugins depend on this one.